colab/test_notebooks.py

```diff
@@ -14,12 +14,7 @@
 
 NOTEBOOKS_TO_SKIP = []
 
-NOTEBOOKS_EXPECTED_TO_FAIL = [
-    # Text classification notebook
-    #   * fails to extract the vocab file on Docker
```

Comment:
Hmmm, this is still failing. Should be fixable though.

Reply:
Can't figure this out and it's not worth my time, unfortunately :/

I think we should be setting TMPDIR or some other environment variable, but I'm not able to debug these notebooks under Docker effectively.